tests/unit/test_statements.py

```diff
@@ -9,4 +9,6 @@ def test_series_name(ro_annual_capiq_stmts: FinancialStatements):
     # Check Cash and Cash Equivalents Name is set. This item includes spaces
     assert stmts.cash.name == "Cash and Cash Equivalents"
     # Check a calculated item name is set
-    assert stmts.nwc.name == "nwc"
+    # nwc is now a first class config item
+    # TODO: Test a different calculated field if they will still exist
```

Comment:
Why shouldn't this still work? Also isn't NWC still a calculated field as it's not in the statement data?

Reply:
I've added nwc to the config file as well as a 'show_on_statement' attribute.
nwc was part of bs/data.py but didn't surface on the statement output.
I thought adding it to the config and not surfacing it on the statements was the best way to handle it for now. in the future, we might have a supplemental or metrics statement that has this and other items (like effective tax rate)

Reply:
added back.